### ISSUE TITLE
Icon request: Search 

### ISSUE THREAD

**Issue by mgailius** [google/material-design-icons#1571]
Perhaps we could have a regular search icon with a shorter handle, similar to the current `Mystery` icon, but of course, without the eye🙏
![image](https://github.com/google/material-design-icons/assets/26027429/cd0e594c-08f2-45a4-8d21-712d15201318)

**Comment by khubaibalam2000** [comment]
Can I work on this issue? Can you please assign me?

**Comment by khubaibalam2000** [comment]
Can I work on this issue ? I already setup the project.

**Comment by tphinney** [comment]
No, sorry you cannot work on it. See the readme. https://github.com/google/material-design-icons/blob/master/README.md

“Please note that Google Fonts does not accept user submissions of finished icon designs! There are fairly strict  guidelines for Material icons, plus Google has upstream source files from which this repo is generated. Therefore, Google does not accept pull requests for icon files (whether new icon suggestions, or fixes for existing icons). Concepts are appreciated—just don’t design SVGs and submit them via pull request.”

**Comment by khubaibalam2000** [comment]
Ok.
